roc/receiver.go

```diff
@@ -30,6 +35,63 @@ func OpenReceiver(rocContext *Context, receiverConfig *ReceiverConfig) (*Receive
 	return (*Receiver)(receiver), nil
 }
 
+func (s *Receiver) Bind(portType PortType, proto Protocol, a *Address) error {
+	errCode := C.roc_receiver_bind(
+		(*C.roc_receiver)(s),
+		(C.roc_port_type)(portType),
+		(C.roc_protocol)(proto),
+		a.raw)
+	if errCode == 0 {
+		return nil
+	}
+	if errCode < 0 {
+		return ErrInvalidArguments // roc_receiver_bind returns negative in 3 cases: invalid args, cannot bind address and not enough resources, maybe we should consider renaming this error to a more generic name?
+	}
+	return ErrInvalidApi
+}
+
+func (r *Receiver) ReadAudioBytes(bytes []byte, samplesSize uint) error {
+	c_roc_frame := C.roc_frame{
+		samples:      (unsafe.Pointer(&bytes[0])),
+		samples_size: (C.ulong)(samplesSize),
+	}
+
+	errCode := C.roc_receiver_read(
+		(*C.roc_receiver)(r),
+		&c_roc_frame,
+	)
+
+	if errCode == 0 {
+		return nil
+	}
+
+	if errCode < 0 {
+		return ErrInvalidArguments // roc_receiver_bind returns negative in 3 cases: invalid args, cannot bind address and not enough resources, maybe we should consider renaming this error to a more generic name?
+	}
+	return ErrInvalidApi
+}
+
+func (r *Receiver) ReadAudioFloats(floats []float32, samplesSize uint) error {
```

Comment:
1. I suggest to rename it to just `ReadFloats` for clarity.
2. `samplesSize` parameter is redundant - we can just use `len(floats)` (like io.Reader does)